crates/solidity/outputs/cargo/tests/src/bindings_assertions/assertions.rs

```diff
@@ -21,34 +21,40 @@ pub enum AssertionError {
     #[error("Duplicate assertion definition {0}")]
     DuplicateDefinition(String),
 
-    #[error("Failed {failed} of {total} bindings assertions: {errors:?}")]
+    #[error("Failed {failed} of {total} bindings assertions:\n{}", display_assertion_errors(.errors))]
     FailedAssertions {
         failed: usize,
         total: usize,
         errors: Vec<String>,
     },
```

Comment:
nit suggestion: you can use debug format here as well:

```suggestion
    #[error("Failed {failed} of {total} bindings assertions:\n{errors:?}")]
    FailedAssertions {
        failed: usize,
        total: usize,
        errors: Vec<String>,
    },
```

Reply:
Yes, but the output format is less readable and an inconvenience when debugging failed assertions. Since it's a `Vec` it would print them separated by commas instead of putting each failure in a new line.

Reply:
I see. Then I can suggest `{errors:#?}`, which will put them on a separate line. But completely up to you.

```rust
let x = vec!["a", "b", "c"];
println!("{x:#?}");
```

```log
[
    "a",
    "b",
    "c",
]
```